python/lsst/ctrl/bps/wms/panda/cmd_line_embedder.py

```diff
@@ -0,0 +1,94 @@
+import os

```

Comment:
You need to put the proper file headers at the top of each file.

Reply:
Done, I put the file header to this file.